### DISCUSSION TITLE
Search results are not displayed when button with link is added at the beginning.

### DISCUSSION THREAD

**Discussion by manjunathreddy1** [microsoft-search/pnp-modern-search#3501] [Q&A]
**Version used**
4.10.1

**Describe the bug**
Search results webpart is not displayed when button with link is added at the beginning but if we try to add it at the end it is properly displayed. Please help to fix this issue.

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Add search verticals web part and three verticals 1.button with link 2. only button 3. only button
2. add search results webpart to the page and configure to show the results when second button is selected
3. now if we select the fist button with link no results webpart is displayed

**Expected behavior**
Results webpart should be displayed

**Screenshots**
![image](https://github.com/microsoft-search/pnp-modern-search/assets/265137/59db53f3-003b-4262-9926-6268e8e30c4e)

**Desktop (please complete the following information):**
 - Chrome and Edge browser

**Comment by wobba** [comment]
Would be good if you can expand a bit more on the actual setup.
